autogen/json2class.cpp

```diff
@@ -1650,34 +1701,59 @@ void file_python_class(const NameDeps &obj, const std::string &filePythonCPP)
    cpp << "         >(),\n";
 
    // python::arg...
-   for (auto &m : minfo)
-      cpp << "         python::arg(\"" << m.varName << "\"),\n";
-   for (auto &c : cinfo)
-      cpp << "         python::arg(\"" << c.varName << "\"),\n";
+   for (auto &m : minfo) {
```

Comment:
I'd considered this autogeneration code to be sort of a hack, but I see it was clear enough (and/or you're smart enough) that you were able to figure out how to work with its data structures to make the changes you needed to make. Good work!

Reply:
I'll take that as a complement ;-)